examples/experimental/koen/CNN plan.ipynb

```diff
@@ -0,0 +1,1058 @@
+{
```

Comment:
<p>would be really cool if this dummy data could be automatically generated based on the input argument variables</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/5455/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>Agreed, I do think this would be a nice stand alone ticket so we might leave it our for now?</p>

Reply:
I think a requirement for this is that our API for calling the forward plans exactly matches the api for calling the torch.nn.Module.forward (I think this is a good idea anyway), right now there is a difference:

SyModule.forward
```
out = self.layer(x=x)[0]
```
torch.nn.Module.forward
```
out = self.layer(x)
```